### DISCUSSION TITLE
Driver electric seat

### DISCUSSION THREAD

**Discussion by Miguelgamito1** [cedricp/ddt4all#884] [Q&A] [answered]
Hello, with the ecu : Identified ECU [MemoSeatPassenger - LDCM]@F7 : DSMU_CMF1_V1.0_20170901T140146.xml 
i can control the passenger seat and do some configurations, but i want to do the same for the driver seat? how can i do it?
thanks for the help.

i have a Renault Talisman

**Comment by KarelSvo** [comment, marked as answer]
DSMU Driver
PSCU Passenger
Talisman Phase 1 CMF1
Talisman Phase 2 C1A

**Comment by Miguelgamito1** [comment]
thanks
